components/vc/front/include/CANIO_componentSpecific.h

```diff
@@ -86,10 +86,9 @@
 #define set_shockpotVoltFL(m,b,n,s) set(m,b,n,s, shockpot_getFLVoltage())
 #define set_shockpotVoltFR(m,b,n,s) set(m,b,n,s, shockpot_getFRVoltage())
 
-#define set_wheelSpeedLinearL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedLinear(WHEEL_FL))
-#define set_wheelSpeedLinearR(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedLinear(WHEEL_FR))
-#define set_wheelSpeedRotationalL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FL))
-#define set_wheelSpeedRotationalR(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FR))
+#define set_wheelSpeedRotationalFL(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FL))
+#define set_wheelSpeedRotationalFR(m,b,n,s) set(m,b,n,s, wheelSpeed_getSpeedRotational(WHEEL_FR))
+#define set_axleFrontRPM(m,b,n,s) set(m,b,n,s, wheelSpeed_getAxleRPM(AXLE_FRONT))
```

Comment:
units are baked into the dbc, no need to put them in the signal name
```suggestion
#define set_axleSpeedFront(m,b,n,s) set(m,b,n,s, wheelSpeed_getAxleRPM(AXLE_FRONT))
```